src/components/popover-manager/popover-manager.e2e.ts

```diff
@@ -142,4 +146,72 @@ describe("calcite-popover-manager", () => {
 
     expect(await popover.getProperty("open")).toBe(false);
   });
+
+  it("autoClose grouped popovers when clicked outside", async () => {
+    const page = await newE2EPage();
+
+    await page.setContent(
+      `
+      <div id="outsideNode">Outside node</div>
+      <calcite-popover-manager auto-close group="test">
+        <calcite-popover id="popover1" reference-element="ref" open>
```

Comment:
> does the popover manager's group prop only work w/ autoClose?

Yes, currently. That's a good point though. I can update the query logic to only match the specific group.

> do we need to worry about supporting multiple groups within a single popover manager?

I don't think so. I don't see a purpose for doing that.

> if there are nested popover-managers (e.g., within higher level web components), there is a slight possibility of duplicate group names. Can popovers managers distinguish between these?

If there are duplicate names, it just means that autoClose will close for either popover manager. I don't see this as an issue. If that is undesirable it would be a good idea to make the group names more unique.

Reply:
> So yes, they only work with autoClose. There's no need to only open popovers with a specific group.

To make it clearer that auto-close is tied to the group, could the prop accept the group name? 

```html
<calcite-popover-manager auto-close><!-- auto close ungrouped --></calcite-popover-manager>

<calcite-popover-manager auto-close="close-me"><!-- auto close 'close-me' group --></calcite-popover-manager>
```

> I don't think so. I don't see a purpose for doing that.

It think it's a naming thing. I find it odd that `group` is restricted to just one and that it only works for auto closing. Would adding a `no-auto-close` (example name) to popovers accomplish the same thing?

```html
<div id="outsideNode">Outside node</div>
<calcite-popover-manager auto-close>
  <calcite-popover no-auto-close id="popover1" reference-element="ref" open>
    Popover1
  </calcite-popover>
  <calcite-popover id="popover2" reference-element="ref2" open>
    Popover2
  </calcite-popover>
  <div id="ref">Button</div>
  <div id="ref2">Button</div>
<calcite-popover-manager>
```


```html
<div id="outsideNode">Outside node</div>
<calcite-popover-manager auto-close>
  <calcite-popover id="popover1" reference-element="ref" open>
    Popover1
  </calcite-popover>
  <calcite-popover id="popover2" no-auto-close reference-element="ref2" open>
    Popover2
  </calcite-popover>
  <div id="ref">Button</div>
  <div id="ref2">Button</div>
<calcite-popover-manager>
```

Reply:
> To make it clearer that auto-close is tied to the group, could the prop accept the group name?

Yeah, we could do that. You are suggesting...

```
<calcite-popover-manager auto-close="mygroup" />
<calcite-popover group="mygroup" /> // will get autoclosed
<calcite-popover group="notmygroup" /> // will not get autoclosed
```

right?

> Would adding a no-auto-close (example name) to popovers accomplish the same thing?

Yes, that would accomplish the same thing so we could go that route too.

---

Part of the issue is that the issue request isn't actually correct. https://github.com/Esri/calcite-components/issues/3701

It says for requested behavior....

> only closes popovers that are inside the popover-manager node or inside the popover-manager node and match the selector

However, the popover manager listens for click events on the window, not its host element. Only the reference elements need to be contained inside the popover manager.

---

@jcfranco These are good questions. I guess I'm not sure what the best solution is.

Reply:
> I see. Can we clarify the requirements in the issue before proceeding?

Yes, I'll verify.

> If you think the manager-less tooltip/popover approach would fix this, we can bump the priority on it.

Yes, I think that might help clear some things up. Can we prioritize it?

Reply:
> Can we prioritize it?

Let's do this! 💪🏼